backend/llm_integration.py

```
@@ -559,38 +705,77 @@ def stream_ai_answer(context: str, question: str, provider: str,
+                        ],
+                        max_tokens=320,
+                        temperature=0.2,
+                        repeat_penalty=1.1,
+                        stream=True,
+                    )
+                    for output in stream:
+                        delta = output['choices'][0].get('delta', {})
+                        token = delta.get('content')
+                        if token:
+                            yield token
+                    return
```

Ruff (0.15.20)

[warning] Consider moving this statement to an `else` block

(TRY300)